bcda/models/models.go

```diff
@@ -347,6 +348,19 @@ type CCLFBeneficiary struct {
 	BlueButtonID string `gorm:"type: text"`
 }
 
+type Suppression struct {
+	gorm.Model
+	HICN                string `gorm:"type:varchar(11);not null"`
+	SourceCode          string `json:"source_code"`
```

Comment:
Should there be a length on the source code field? It has length 5 in the ICD (not clear if it's fixed or variable though).

Reply:
good eyes! Yep I saw the same thing but the ICD also gives 2 examples of valid values...kinda contradicting their own ICD. They show 2 examples: "1800" and "1-800" one being length 4 and one being length 5, so I just went with a varchar...but maybe there's a better way. thoughts?

Reply:
Aha. In that case, `varchar(5)` probably works: variable length, max 5.

Reply:
nice!